runtime/nsd_ossl.c

```diff
@@ -78,6 +78,7 @@ DEFobjCurrIf(nsd_ptcp)
 	#define RSYSLOG_BIO_number_read(SSLBIO) SSLBIO->num
 	#define RSYSLOG_BIO_number_written(SSLBIO) SSLBIO->num
 #endif
+#define OSSL_VERIFY_DEPTH 100
```

Comment:
I guess this can be removed?

Reply:
We need the default here as we set the verify_depth default here for the existing API call which was hardcoded to 2 before.

Reply:
Removed need of OSSL_VERIFY_DEPTH , it should be fine now